### PR TITLE
myChangedimes

### PR DESCRIPTION
**Pull request by Manojk243**

### CODE REVIEW

**Comment by erpbizsol** [comment]
plz Check Manoj

File: BizsolERP/BizsolERPMain/Controllers/HomeController.cs [hunk 1]
```diff
@@ -15,7 +15,7 @@ public HomeController(ILogger<HomeController> logger)
 
         public IActionResult Index()
         {
-            return View();
+            return View();/// mamisjsjklaisks sajssisj
```
[the remark this answers is not in the document]
Comment (Manojk243) [reply BizsolERP/BizsolERPMain/Controllers/HomeController.cs:18]: ok i will changes

File: BizsolERP/BizsolERPMain/Controllers/HomeController.cs [hunk 2]
```diff
@@ -15,7 +15,7 @@ public HomeController(ILogger<HomeController> logger)
 
         public IActionResult Index()
         {
-            return View();
+            return View();/// mamisjsjklaisks sajssisj 
         }
```
Comment (erpbizsol) [inline BizsolERP/BizsolERPMain/Controllers/HomeController.cs:20]: whay this here
Screenshot 2023-05-04 115331

File: BizsolERP/BizsolERPMain/Controllers/HomeController.cs [hunk 3]
```diff
@@ -15,7 +15,7 @@ public HomeController(ILogger<HomeController> logger)
 
         public IActionResult Index()
```
Comment (erpbizsol) [inline BizsolERP/BizsolERPMain/Controllers/HomeController.cs:16]: his Manoj

File: BizsolERP/BizsolERPMain/Controllers/HomeController.cs (hunk 2 above)
Comment (Manojk243) [reply BizsolERP/BizsolERPMain/Controllers/HomeController.cs:20]: done

File: BizsolERP/BizsolERPMain/Controllers/HomeController.cs [hunk 4]
```diff
@@ -15,7 +15,7 @@ public HomeController(ILogger<HomeController> logger)
...
         {
-            return View();
+            return View();/// mamisjsjklaisks sajssisj 
         }
```
Comment (erpbizsol) [inline BizsolERP/BizsolERPMain/Controllers/HomeController.cs:19]: why this here

File: BizsolERP/BizsolERPMain/Controllers/HomeController.cs (hunk 1 above)
[the remark this answers is not in the document]
Comment (Manojk243) [reply BizsolERP/BizsolERPMain/Controllers/HomeController.cs:18]: as

**Pull request merged**